packages/cadl-ranch-specs/http/type/model/visibility/main.tsp

```diff
@@ -59,14 +82,14 @@ model ReadOnlyModel {
   ```
   """)
 @get
-op getModel(@body input: VisibilityModel): {
+op getModel(@body input: QueryModel): {
```

Comment:
if we want to test query parameters with readonly, I think we should be more clear and add another test for it, that way the scenario name is accurate

Reply:
Operation GetModelWithQuery() Added.